lib/api/BackbeatAPI.js

```diff
@@ -113,6 +135,114 @@ class BackbeatAPI {
         });
     }
 
+    /**
+     * Get replication backlog in bytes and count
+     * @param {function} cb -callback(error, data)
+     * @return {undefined}
+     */
+    backlog(cb) {
+        async.series([
+            next => this._queryStats(OPS, next),
+            next => this._queryStats(BYTES, next),
+        ], (err, res) => {
+            if (err) {
+                this._logger.trace('error occurred getting backlog');
+                return cb(errors.InternalError);
+            }
+            const data = {
+                backlog: {
+                    details: 'Replication backlog in bytes and count',
+                    count: res[0].requests,
+                    bytes: res[1].requests,
+                },
+            };
+            return cb(null, data);
+        });
+    }
+
+    /**
+     * Get completed replicated stats by bytes and count
+     * @param {function} cb - callback(error, data)
+     * @return {undefined}
+     */
+    completions(cb) {
+        async.series([
+            next => this._queryStats(OPS_DONE, next),
+            next => this._queryStats(BYTES_DONE, next),
+        ], (err, res) => {
+            if (err) {
+                this._logger.trace('error getting replication completions');
+                return cb(errors.InternalError);
+            }
+            const data = {
+                completions: {
+                    details: 'Replciations completed in bytes and count in ' +
```

Comment:
typo: Replications

I would also rephrase to something more explicit for what is count and bytes, so something like

Number of completed replication operations (count) and number of bytes transferred (bytes) in the last ... seconds

Reply:
I've switched to converting total bytes in MBs to keep things uniform. Also, instead of naming the property "bytes" or "MBs", I've just switched to using "size"

Lmk if that sounds good, or if you prefer different property names 👍